### ISSUE TITLE
Can it be used in .net8?

### ISSUE THREAD

**Issue by gaesar1** [xarial/xcad#123]
When I use .net8, the registry doesn't respond, but .net 7works fine.

**Comment by dubeg** [comment]
Is there a reason it doesn't work yet, or is it merely a matter of updating the templates / libraries to net8/9?

**Comment by foreachthing** [comment]
Yeah, I have the same issue: .net8.0 with SolidWorks 2024 won't work anymore.

**Comment by artem1t** [comment]
Public [Alpha version](https://www.nuget.org/packages/Xarial.XCad.SolidWorks/0.9.0-alpha.5200#supportedframeworks-body-tab) is compatible with .NET8 and .NET9

**Comment by foreachthing** [comment]
@artem1t, thanks! I didn't know about the Alpha version. 👍🏼
